Documentation/gettingstarted/k8s-install-helm.rst

```diff
@@ -174,10 +174,11 @@ Install Cilium
           Kubernetes worker node than the ENI limit, but means that pod
           connectivity to resources outside the cluster (e.g., VMs in the VPC
           or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use
-          the VPC IP address of the Kubernetes worker node.  Excluding the
-          lines for ``eni.enabled=true``, ``ipam.mode=eni`` and
-          ``tunnel=disabled`` from the helm command will configure Cilium to
-          use overlay routing mode (which is the helm default).
+          the VPC IP address of the Kubernetes worker node.
+
+            * Excluding the lines for ``eni.enabled=true``, ``ipam.mode=eni`` and ``tunnel=disabled`` from the helm command will configure Cilium to use overlay routing mode (which is the helm default).
+            * Flush iptables to clear any iptables configurations done by VPC CNI
+            * Restart ``kube-proxy`` pod to reconfigure iptables
```

Comment:
Reading the paragraph only, it's not clear to me how this list should be interpreted. We should probably introduce it with something like “To set up this overlay mode, follow the steps below:” (then maybe used an ordered list (with numbers)?).

Also I would like to have the exact commands to run for the last two steps.

Reply:
Thanks for your opinion. I will rewrite the documentation later.

Reply:
Here are the last two commands:
1. `iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X`
2. `kubectl delete pod -l k8s-app=kube-proxy -n kube-system`